ddtrace/contrib/logging/patch.py

```diff
@@ -73,7 +74,13 @@ def _w_makeRecord(func, instance, args, kwargs):
         span = _get_current_span(tracer=ddtrace.config.logging.tracer)
 
     if span:
-        setattr(record, RECORD_ATTR_TRACE_ID, str(span.trace_id))
+        if span.trace_id > _MAX_UINT_64BITS and not ddtrace.config._128_bit_trace_logging_enabled:
+            # Ensure 64 bit trace id is logged (instead of the 64bit version)
+            trace_id = str(span._trace_id_64bits)
+        else:
+            trace_id = str(span.trace_id)
+
+        setattr(record, RECORD_ATTR_TRACE_ID, str(trace_id))
```

Comment:
```suggestion
```

I plan on reverting this change.

Logs ingestion will break if a 128bit trace id is set. As a workaround DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED was defined and will be set for False for the foreeable future. When logs ingestion is compatible with 128bit trace ids then we can remove this flag and log the full 128bit trace id. I am not sure if we want this hack but I left it in for now.